src/frontend/index.ts

```diff
@@ -66,25 +71,40 @@ export async function createFlowrWindow(flowrStore: Store<IFlowrStore>): Promise
       contextIsolation: false,
       partition: 'persist:flowr', // needed to display webcam image
       preload: buildPreloadPath('exportNode.js'),
-      enableRemoteModule: true, // TODO: FLOW-8215
     },
   })
 
   const mainWindow = new FlowrWindow(flowrStore, opts)
 
+  function loadConfigPage() {
+    mainWindow.loadURL(defaultUrl)
+      .catch(e => {
+        console.error('Failed to load default configuration page.... sorry but can\'t do anything else for you', e)
+      })
+  }
+
+  function loadFlowr(flowrURL: string): void {
+    mainWindow.loadURL(flowrURL)
+      .catch((e: Error) => {
+        console.warn('Error loading flowr window', e)
+        lastError = e.message
+        loadConfigPage()
```

Comment:
TODO: do not load config page, it could be a backdoor. Retry instead

Reply:
bump